string segment_id
soma_llsd_msgs/Observation[] observations
---
bool result
